docker/resource-provider/Dockerfile

```diff
@@ -1,28 +1,56 @@
-FROM golang:latest as base
+FROM ubuntu:latest
 WORKDIR /usr/src/app
 
-ARG doppler_config=dev
-ARG network=dev
-ARG DOPPLER_TOKEN_BACALHAU
-ARG DOPPLER_TOKEN_RESOURCE_PROVIDER
+# Default environment variables
+ENV LOG_TYPE=json
+ENV LOG_LEVEL=debug
+ENV HOME=/app/lilypad
+ENV BACALHAU_SERVE_IPFS_PATH=/data/ipfs
+ENV OFFER_GPU=1
 
-COPY . .
-RUN go mod download && go mod verify
-RUN go build -v .
-RUN go install
+# Install necessary dependencies
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    git \
+    curl \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
 
-RUN (curl -Ls --tlsv1.2 --proto "=https" --retry 3 https://cli.doppler.com/install.sh || wget -t 3 -qO- https://cli.doppler.com/install.sh) | sh
+# Installing Bacalhau
+RUN cd /tmp && \
+    wget https://github.com/bacalhau-project/bacalhau/releases/download/v1.3.0/bacalhau_v1.3.0_linux_amd64.tar.gz && \
```

Comment:
Could we upgrade this to Bacalhau `v1.3.2`? @AquiGorka has been using `v1.3.2` without issue.

Reply:
Yes - upgrade done